core/src/main/scala/chisel3/Num.scala

```diff
@@ -5,6 +5,7 @@ package chisel3
 import chisel3.internal.sourceinfo.SourceInfoTransform
 
 import scala.language.experimental.macros
+import scala.math.BigDecimal.RoundingMode._
```

Comment:
Nit: I'd avoid the wildcard import here and just do:

```suggestion
import scala.math.BigDecimal.RoundingMode.RoundingMode
```